setup.cfg

```diff
@@ -71,3 +71,8 @@ ignore =
     sklearn/utils/_seq_dataset.pxd
     sklearn/utils/_weight_vector.pyx
     sklearn/utils/_weight_vector.pxd
+
+[codespell]
+skip = ./.git,./doc/themes/scikit-learn-modern/static/js,./sklearn/feature_extraction/_stop_words.py,./doc/_build,./doc/auto_examples,./doc/modules/generated
```

Comment:
May we add `./.mypy_cache` here too?

Reply:
I have added `./.mypy_cache`.

Using two exclusion mechanisms is complicated and error-prone, I agree with that. However, I'm worried that some of the words that we ignore will shadow very common typos, the kind you see in any large project. The most striking example is `teh`.

Reply:
I was hoping that at least files `v*.rst` wouldn't change much.